dash/backend/src/modules/auth/auth.module.ts

```diff
@@ -13,8 +13,6 @@ import {LdapFactory} from './factories/LdapFactory';
 import LoggedInUserFactory from '../../providers/LoggedInUserFactory';
 import {ResetPasswordService} from './services/reset-password.service';
 import {ResetPasswordDao} from './dao/reset-password.dao';
-import {AppSettingsService} from '../settings/services/app-settings.service';
```

Comment:
Why are we removing this?

Reply:
Is it just dead code?

Reply:
When I tried to boot up locally, I was prompted to remove so (not allowed to be in certain modules). It might be from the nestjs upgrade.